src/XCCDF/public/xccdf_session.h

```diff
@@ -42,6 +42,19 @@
 struct xccdf_session;
 
 /**
+ * Loading flags for XCCDF session
+ * @memberof xccdf_session
+ * The flags set which components will be loaded in XCCDF session.
+ */
+typedef enum {
+	XCCDF_SESSION_LOAD_NONE = 0,
```

Comment:
Thank you. What wouldn't be misleading? XCCDF_SESSION_LOAD_ONLY_XCCDF or XCCDF_SESSION_LOAD_NO_COMPONENT or something else?